src/cascade/dismod/constants.py

```diff
@@ -151,12 +151,12 @@ class PriorKindEnum(Enum):
          "mulcov", "node", 'nslist_pair', "nslist", "option",
          "prior", "rate", "smooth_grid", "smooth", "time",
          "weight_grid", "weight"],
-        ["age_avg", "var", "data_subset", "start_var", "scale_var"]),
+        ["age_avg", "var", "data_subset", "start_var", "scale_var", "truth_var"]),
```

Comment:
Truth_var wasn't listed by Dismod-AT documentation among affected tables, but it's in the affected tables, so this adds it.